README.md

```diff
@@ -278,6 +278,50 @@ API. The commented interface is available in
 and
 [http](src/python/library/tritonclient/http/__init__.py).
 
+### GRPC Options
+
+#### SSL/TLS
+
+The client library allows communication across a secured channel using gRPC protocol.
+
+For C++ client, see `SslOptions` struct that encapsulates these options in [grpc_client.h](src/c%2B%2B/library/grpc_client.h).
```

Comment:
I think @tanmayv25 brought this up but we should not point to specific sections of our code in the readme